submarine-dist/src/assembly/distribution.xml

```diff
@@ -185,6 +181,11 @@
         <!-- atomix & netty-4.1.27.Final already shade in submarine-commons-cluster-${project.version}-shade.jar -->
         <exclude>atomix-*.jar</exclude>
         <exclude>netty-*-4.1.27.Final*.jar</exclude>
+        <exclude>submarine-cloud/bin/**</exclude>
+        <exclude>dev-support/submarine-installer/package/hadoop/yarn/lib/native/**</exclude>
```

Comment:
The "<exclude>submarine-cloud/bin/**</exclude>" and "<exclude>dev-support/submarine-installer/package/hadoop/yarn/lib/native/**</exclude>" seems should be put in the "submarine-dist/src/assembly/src-distribution.xml"

Reply:
okay, got it.